<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="RenderViewWidget" name="renderViewWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSlider" name="timeNowSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="timeNowSpinBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>915</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLol_MIC_stinkt">
    <property name="title">
     <string>Offend</string>
    </property>
    <addaction name="actionNIX_riecht_nicht_so_gut"/>
    <addaction name="actionDR_QM_riecht_besser_als_NIX_seine_Oma"/>
    <addaction name="actionNR4_hat_das_Stinken_erfunden"/>
    <addaction name="actionGrenzdevil_hat_Lauras_F_en_Stinken_beigebracht"/>
    <addaction name="actionMIC_kauft_ranzigem_Fu_den_Schneid_ab"/>
    <addaction name="actionDaDummy_riecht_von_allen_hier_am_Besten"/>
   </widget>
   <addaction name="menuLol_MIC_stinkt"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>300</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>KONTROL 210</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="nanoKontrol2Widget" name="nanoKontrol2" native="true"/>
     </item>
     <item>
      <widget class="apc40Mk2Widget" name="apc40Mk2" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNIX_riecht_nicht_so_gut">
   <property name="text">
    <string>NIX riecht nicht so gut</string>
   </property>
  </action>
  <action name="actionDR_QM_riecht_besser_als_NIX_seine_Oma">
   <property name="text">
    <string>DR. QM riecht besser als NIX seine Oma</string>
   </property>
  </action>
  <action name="actionNR4_hat_das_Stinken_erfunden">
   <property name="text">
    <string>NR4 hat das Stinken erfunden</string>
   </property>
  </action>
  <action name="actionGrenzdevil_hat_Lauras_F_en_Stinken_beigebracht">
   <property name="text">
    <string>Grenzdevil hat Lauras Füßen Stinken beigebracht</string>
   </property>
  </action>
  <action name="actionMIC_kauft_ranzigem_Fu_den_Schneid_ab">
   <property name="text">
    <string>MIC kauft ranzigem Fuß den Schneid ab</string>
   </property>
  </action>
  <action name="actionDaDummy_riecht_von_allen_hier_am_Besten">
   <property name="text">
    <string>DaDummy riecht von allen hier am Drittbesten</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RenderViewWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>renderviewwidget.h</header>
  </customwidget>
  <customwidget>
   <class>nanoKontrol2Widget</class>
   <extends>QWidget</extends>
   <header>nanokontrol2Widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>apc40Mk2Widget</class>
   <extends>QWidget</extends>
   <header>apc40mk2widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
